karma.conf.cjs

```diff
@@ -0,0 +1,62 @@
+// Karma configuration file, see link for more information
+// https://karma-runner.github.io/1.0/config/configuration-file.html
+process.env.CHROME_BIN = require('puppeteer').executablePath();
+
+// test/karma.conf.js
+module.exports = config => {
+  config.set({
+    frameworks: ['jasmine', 'webpack'],
+    basePath: '',
+    type: 'module',
+    files: [
+      {pattern: 'js/*.ts', type:'module'},
```

Comment:
Well, technically no, but in this instance, unfortunately, the way these tests work is that they grab all the TS files, compile them in memory, and then run the tests against the binary (no artifact files are created as a result). Good point that they don't run directly against `geemap/static`, and I'm not entirely sure if we can get the TS files to reference them directly? For now, it's nice to be able to compile everything in the same environment rather than a mishmash of TS and JS. We can always manually test the generated code in local notebooks!